src/app/views/main/main.component.html

```diff
@@ -1,28 +1,40 @@
 <div class="container">
 
-  <div class="row">
-    <div class="testTextContainer col-md-12">
-      <span>{{ currentText }}</span>
+  <div class="row justify-content-between">
+    <div class="rewriteContainer col-md-7">
+      <div class="row">
+        <div class="testTextContainer col-md-12">
+          <span *ngFor='let text of this.currentTextArray' [class.complete]="text.completed">{{ text.word }} </span>
+        </div>
+      </div>
+
+      <div class="row">
+        <input #rewriteInput on-keyup='checkWord($event.keyCode, rewriteInput.value)' [disabled]=!isTestInProgress type="text" placeholder="Type text here" class="form-control col-md-12"
+        />
+      </div>
+    </div>
+    <div class="appConsole col-md-4">
+      <ul class='consoleListContainer'>
+        <li *ngFor='let alert of consoleService.consoleArray.reverse()' class='consoleMessage'>
```

Comment:
Templatka raczej nie powinna odwoływać się wprost do serwisu. Stworzyłbym w komponencie metodę `getAlerts()`, w której mielibyśmy `return this.consoleService.consoleArray.reverse()`. Lub spróbował z observable.